lib/rules/no-multi-spaces.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @fileoverview This rule warns about the usage of extra whitespaces between attributes
+ * @author Armano
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'This rule warns about the usage of extra whitespaces between attributes',
+      category: 'Stylistic Issues',
+      recommended: false
+    },
+    fixable: 'whitespace',  // or "code" or "whitespace"
+    schema: []
+  },
+
+  /**
+   * @param {RuleContext} context - The rule context.
+   * @returns {Object} AST event handlers.
+   */
+  create (context) {
+    const sourceCode = context.getSourceCode()
+
+    // ----------------------------------------------------------------------
+    // Public
+    // ----------------------------------------------------------------------
+
+    return {
+      Program (node) {
+        // TODO: Check `context.parserServices.getTemplateBodyTokenStore` exists or not.
+        const tokenStore = context.parserServices.getTemplateBodyTokenStore()
+        const tokens = tokenStore.getTokens(node.templateBody, { includeComments: true })
+
+        let prevToken = tokens.shift()
+        for (const token of tokens) {
+          if (sourceCode.isSpaceBetweenTokens(prevToken, token)) {
+            const text = sourceCode.getText(token, (token.range[0] - prevToken.range[1] + 1), 0)
+
+            const match = text.match(/([^\r\n\t\s])([ \t]+)([>]?)([\n\r]?)/)
+            if (!match) {
+              prevToken = token
+              continue // there is no errors
+            }
+
+            const spaces = match[2].length
+            const requiredSpaces = match[3] === '>' || match[4] !== '' ? 0 : 1
+
+            if (spaces > requiredSpaces) {
+              context.report({
+                node: token,
+                loc: {
+                  start: {
+                    line: prevToken.loc.end.line,
+                    column: prevToken.loc.end.column + requiredSpaces
+                  },
+                  end: {
+                    line: prevToken.loc.end.line,
+                    column: prevToken.loc.end.column + spaces
+                  }
+                },
+                message: 'Extra whitespace detected.',
+                fix: (fixer) => fixer.removeRange([prevToken.range[1] + requiredSpaces, prevToken.range[1] + spaces])
```

Comment:
in this case:
```
fix: (fixer) => fixer.replaceTextRange([prevToken.range[1], prevToken.range[1] + spaces], ' '),
```

is going to be valid cus there can be \n\r symbold before space